releasenotes/notes/ignore-zombie-process-config-cf78b9d48d34c702.yaml

```diff
@@ -0,0 +1,14 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Added config that allows user to toggle on and off the collection of Zombie processes in the agent.
+fixes:
+  - |
+    Prevention of OOMs due to excessive collection of Zombie processes.
```

Comment:
```suggestion
    Prevention of OOMs due to excessive collection of zombie processes.
```

Reply:
Nit: "excessive collection" sounds off to me. Maybe "Prevention of OOMs when collecting a large number of zombie processes."